### PR TITLE
Freelancer responsive fix

### PR DESCRIPTION
**Pull request by paviramu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1121,11 +1121,12 @@
             height: 16px;
             width: 16px;
             margin-left: 8px;
-            
+
```
Comment (Ajithonline) [inline index.html]: Remove this spacing

File: index.html [hunk 2]
```diff
@@ -1121,11 +1121,12 @@
...
             width: 16px;
             margin-left: 8px;
-            
+
         }
-       
+
         textarea {
             height: 48px;
+
```
Comment (Ajithonline) [inline index.html]: Space

File: index.html [hunk 3]
```diff
@@ -1121,11 +1121,12 @@
...
         }
 
         pre {
```
Comment (Ajithonline) [inline index.html]: Delete if not needed

File: index.html [hunk 4]
```diff
@@ -1169,9 +1170,9 @@
 
         }
 
-         .checkbox-modal{
+        .checkbox-modal {
             font-size: 12px;
-         }
+        }
 
         .checkboxes label span {
```
Comment (Ajithonline) [inline index.html:1146]: Remove if not needed

**Comment by Ajithonline** [comment]
Test

**Review by Ajithonline — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (paviramu) [reply index.html]: Removed the space

File: index.html (hunk 2 above)
Comment (paviramu) [reply index.html]: removed the space

File: index.html (hunk 3 above)
Comment (paviramu) [reply index.html]: removed the style for pre since it is not used

File: index.html (hunk 4 above)
Comment (paviramu) [reply index.html:1146]: removed the space

**Pull request merged**